astropy/io/misc/tests/test_hdf5.py

```diff
@@ -395,7 +401,7 @@ def test_preserve_serialized(tmpdir):
     assert t1.meta == t2.meta
 
 
-@pytest.mark.skipif('not HAS_H5PY')
+@pytest.mark.skipif('not HAS_H5PY' or 'not HAS_YAML')
```

Comment:
Except that at the moment, since part of the expression is in a string, it will not work correctly (``'a' or 'b'`` is ``'a'``). For now, I think we use strings in most places so I'd recommend just switching to using a single string.